### DISCUSSION TITLE
超链接的下划线是常显了吗？

### DISCUSSION THREAD

**Discussion by nibazshab** [vuepress/ecosystem#230] [General]
以前不是只有鼠标碰到，才会显示下划线的么？最近一个多月一直都是常显的

![image](https://github.com/user-attachments/assets/0e8c134e-f19e-43ef-bf9e-5499ea6ed593)
![image](https://github.com/user-attachments/assets/c865be5a-6f1a-4e30-b6da-f1d870dd981c)

**Comment by Mister-Hope** [comment]
是这样的，如果你不喜欢你可以新增自定义样式覆盖
